src/main/resources/db/changelog/changes/02-create-role-table.yaml

```diff
@@ -0,0 +1,32 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-role-table
+      author: Ihor Romaniuk
+      objectQuotingStrategy: QUOTE_ONLY_RESERVED_WORDS
```

Comment:
why do you need it? the same for other scripts

Reply:
it's auto generated

Reply:
You write your scripts by yourself, it mustn't be auto generated)